frontend/src/pages/modelServing/ModelServingRoutes.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
+import { Navigate, Routes, Route } from 'react-router-dom';
+import ModelServingContextProvider from './ModelServingContext';
+import ModelServingGlobal from './screens/global/ModelServingGlobal';
+
+const ModelServingRoutes: React.FC = () => {
+  return (
```

Comment:
I was gonna. say this is not needed, since the routes in dsg are handled by the project and the global view displays in a single page everything, but then I remember we have the [stats view](https://www.sketch.com/s/113593f8-5970-49d6-a352-709b07639127/a/rbZ7oQE), that could be controlled by url.

Reply:
Update, this is not a review, just thinking out loud.